packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -249,6 +264,13 @@ export default async function handleChildrenEventTypes({
                 : {
                     deleteMany: {},
                   },
+            metadata: {
+              ...(eventType.metadata as Prisma.JsonObject),
+              ...(metadata?.multipleDuration && "length" in unlockedFieldProps
+                ? { multipleDuration: metadata.multipleDuration }
+                : {}),
+              ...(metadata?.apps && "apps" in unlockedFieldProps ? { apps: metadata.apps } : {}),
+            },
```

Comment:
Currently, if any fields are unlocked for a child eventType and specific values are set, they are replaced by the parent eventType's metadata values when updating.

Reply:
why don't you do the same for other fields?

Reply:
All the fields aren't saved inside the metadata and are not overwritten. These two fields are the most common and high priority and shouldn't be overwritten